src/base/registrations/registrar.ts

```diff
@@ -99,44 +111,40 @@ export async function getRegistration(name: string, config: Config, resolver?: E
   };
 }
 
-export async function getAvatar(name: string, config: Config, resolver?: EnsResolver): Promise<string | null> {
+export async function getAvatar(name: string, config: Config, resolver?: EnsResolver): Promise<string | undefined> {
   name = name.toLowerCase();
 
   resolver = resolver ?? await config.provider.getResolver(name);
   if (! resolver) {
-    return null;
+    return;
   }
   return resolver.getText('avatar');
 }
 
-export async function getSeedHost(name: string, config: Config, resolver?: EnsResolver): Promise<string | null> {
+export async function getAnchorsAccount(name: string, config: Config, resolver?: EnsResolver): Promise<string | undefined> {
```

Comment:
Here we also will have an issue since `resolver.getText('anchors')` returns `Promise<string | null>`